docs/iot/tutorials/adc.md

```diff
@@ -0,0 +1,120 @@
+
+## Prerequisites
+
+- Raspberry Pi (2 or greater) with [Raspberry Pi OS installed](https://www.raspberrypi.org/documentation/installation/installing-images/README.md)
+- [MCP3008](https://www.microchip.com/wwwproducts/MCP3008) analog-to-digital converter
+- Three-pin potentiometer
+- Breadboard
+- Jumper wires
+- Raspberry Pi GPIO breakout board (optional/recommended)
+- [.NET SDK](https://dotnet.microsoft.com/download) version 5.0.100 or later
+
+[!INCLUDE [prepare-pi-spi](../includes/prepare-pi-spi.md)]
+
+## Prepare the hardware
+
+Use the hardware components to build the circuit as depicted in the following diagram:
+
+:::image type="content" source="../media/rpi-trimpot_spi-thumb.png" alt-text="A Fritzing diagram showing a circuit with an MCP3008 ADC and a potentiometer" lightbox="../media/rpi-trimpot_spi.png":::
+
+The MCP3008 uses Serial Peripheral Interface (SPI) to communicate. The following are the connections from the MCP3008 to the Raspberry Pi and potentiometer:
+
+<!--markdownlint-disable MD033 -->
+- V<sub>DD</sub> to 3.3V (shown in red)
+- V<sub>REF</sub> to 3.3V (red)
+- AGND to ground (black)
+- CLK to SCLK (orange)
+- D<sub>OUT</sub> to MISO (orange)
+- D<sub>IN</sub> to MOSI (orange)
+- CS/SHDN to CE0 (green)
+- DGND to ground (black)
+- CH0 to variable (middle) pin on potentiometer (yellow)
+
+Supply 3.3V and ground to the outer pins on the potentiometer. Order is unimportant.
+
+Refer to the following pinout diagrams as needed:
+
+| MCP3008  | Raspberry Pi GPIO |
+|----------|-------------------|
+| :::image type="content" source="../media/mcp3008-diagram-thumb.png" alt-text="A diagram showing the pinout of the MCP3008" lightbox="../media/mcp3008-diagram.png"::: | :::image type="content" source="../media/gpio-pinout-diagram-thumb.png" alt-text="A diagram showing the pinout of the Raspberry Pi GPIO header. Image courtesy Raspberry Pi Foundation." lightbox="../media/gpio-pinout-diagram.png":::<br />[Image courtesy Raspberry Pi Foundation](https://www.raspberrypi.org/documentation/usage/gpio/).
+ |
+
+[!INCLUDE [gpio-breakout](../includes/gpio-breakout.md)]
+
+## Create the app
+
+Complete the following steps in your preferred development environment:
+
+1. Create a new .NET Console App using either the [.NET CLI](/dotnet/core/tools/dotnet-new) or [Visual Studio](/dotnet/core/tutorials/with-visual-studio). Name it *AdcTutorial*.
+
+    ```dotnetcli
+    dotnet new console -o AdcTutorial
+    ```
+
+1. [!INCLUDE [tutorial-add-packages](../includes/tutorial-add-packages.md)]
+1. Replace the contents of *Program.cs* with the following code:
+
+    ```csharp
+    using System;
+    using System.Device.Spi;
+    using System.Threading;
+    using Iot.Device.Adc;
+
+    var hardwareSpiSettings = new SpiConnectionSettings(0, 0);
+
+    using SpiDevice spi = SpiDevice.Create(hardwareSpiSettings);
+    using Mcp3008 mcp = new Mcp3008(spi);
```

Comment:
In cases like this, it's obvious what the type is. I recommend using `var` in such a situation.


```suggestion
    using var mcp = new Mcp3008(spi);
```